### ISSUE TITLE
BigGAN: train a good ImageNet model for release

### ISSUE THREAD

**Issue by gwern** [tensorfork/tensorfork#24]
Per #23, releasing a large pretrained model is a powerful enabler. People have not been using BigGAN for photographic projects despite the very fun images based on ImageNet (see [Artbreeder](https://artbreeder.com/)) because the ImageNet BigGAN is not fully available, crippling any hope of transfer learning, and StyleGAN is fundamentally unable to produce reasonably high quality ImageNet images (#21).

We, however, have the TPU pods to train an ImageNet BigGAN ourselves. This would be a public service, and if we use it to test our various changes like flood loss, Evonorm, and SimCLR, would give us an objective metric like FID to compare with the BigGAN publications and assure us we're at least doing it mostly right. Skylion has already trained a few smaller ImageNet BigGANs, so a 512px BigGAN run should be perfectly feasible. If we get gradient checkpointing (#15) and progressive growing (#17), we could even go it one better and release a *1024px* BigGAN. People would go nuts for that.

However, right now we do not have any confidence in our ability to do transfer learning with BigGANs due to their instability (#23), so there is no point in training an ImageNet BigGAN just yet. (We are definitely training Ganbooru regardless of whether transfer learning works, so we should get Ganbooru done, prototype transfer learning on that, and then if we figure out how to do that, then it'll be worthwhile to train an ImageNet BigGAN. Heck, perhaps we could even initialize it from Ganbooru!)
